test/src/org/exist/xmldb/ResourceTest.java

```diff
@@ -254,7 +254,6 @@ public void queryRemoveResource() throws XMLDBException {
 
         for (ResourceIterator iterator = rs.getIterator(); iterator.hasMoreResources();) {
             Resource r = iterator.nextResource();
-            System.err.println("Resource id=" + r.getId() + " xml=" + r.getContent());
```

Comment:
no fail() replacement ?

Reply:
It was not previously considered a fail. I suspect, it was previously meant to say `System.out.println`, but well both in tests apart from debugging is bad.

Reply:
@dizzzz We good?

Reply:
yes, sorry.